src/hooks/useGetInstructorsFacetInfinite.js

```diff
@@ -20,15 +20,13 @@ const useGetInstructorsFacetInfinite = (searchString, inView) => {
     };
   }, [searchString]);
   const fetchInstructors = async ({ pageParam = 1 }) => {
+    const { LMS_BASE_URL } = getConfig();
+    const url = `${LMS_BASE_URL}/admin-console/api/instructor-list/?page=${pageParam}&search_query=${debouncedSearchValue}`;
```

Comment:
Please use axios Params instead of not so clean string interpolation.
in this case:

`axios.get('/api', {
  params: {
    page: pageParam,
    search_query: debouncedSearchValue,
  }
});`